packages/smooth_app/lib/temp/product_extra.dart

```diff
@@ -0,0 +1,33 @@
+// Package imports:
+import 'package:openfoodfacts/model/Attribute.dart';
+import 'package:openfoodfacts/model/AttributeGroup.dart';
+import 'package:openfoodfacts/model/Product.dart';
+
+class ProductExtra {
```

Comment:
so we won't have a ProductExtra class in the end correct? we will just make getAttribute a method of the existing Product class in openfoodfacts-dart?

Reply:
Exactly.